src/braket/circuits/ascii_circuit_diagram.py

```diff
@@ -412,9 +404,7 @@ def _create_output(
     def _build_map_control_qubits(item: Instruction, control_qubits: QubitSet) -> dict(Qubit, int):
         control_state = getattr(item, "control_state", None)
         if control_state is not None:
-            map_control_qubit_states = {
-                qubit: state for qubit, state in zip(control_qubits, control_state)
-            }
+            map_control_qubit_states = dict(zip(control_qubits, control_state))
```

Comment:
This could be automatically fixed if we wanted to enable it but I opted to leave that off and manually fix that. The iterable was flagged as unchanged and thus suggested this change.